operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/Controller.java

```diff
@@ -281,4 +281,8 @@ public void stop() {
       eventSourceManager.stop();
     }
   }
+
+  public DependentResourceManager<R> getDependents() {
```

Comment:
Do we really need to expose `DependentResourceManager` here? Can we just expose the dependents directly?

Reply:
So this is just a getter for that in the controller. For general purpose. Controller is not exposed to the user so don't think this is particularly bad.

Reply:
I'd rather not expose this class if possible since I consider it as an internal class.